src/Storefront/Resources/views/storefront/component/listing/filter/filter-range.html.twig

```diff
@@ -1,26 +1,40 @@
 {% set filterItemId = 'filter-' ~ name ~ '-' ~ random() %}
 
-{% if currencySymbol is not defined %}
-    {% set currencySymbol = page.header.activeCurrency.symbol %}
-{% endif %}
-
 {% if filterRangeActiveMinLabel is not defined %}
-    {% set filterRangeActiveMinLabel = 'listing.filterRangeActiveMinLabel'|trans|sw_sanitize %}
+    {% set filterRangeActiveMinLabel = 'listing.filterRangeActiveMinLabel'|trans({'%displayName%': displayName})|sw_sanitize %}
 {% endif %}
 
 {% if filterRangeActiveMaxLabel is not defined %}
-    {% set filterRangeActiveMaxLabel = 'listing.filterRangeActiveMaxLabel'|trans|sw_sanitize %}
+    {% set filterRangeActiveMaxLabel = 'listing.filterRangeActiveMaxLabel'|trans({'%displayName%': displayName})|sw_sanitize %}
 {% endif %}
 
 {% if filterRangeErrorMessage is not defined %}
     {% set filterRangeErrorMessage = 'listing.filterRangeErrorMessage'|trans|sw_sanitize %}
 {% endif %}
 
+{# For backward compatibilty #}
+{% if unit is not defined %}
+    {% if currencySymbol is not defined %}
+        {% set currencySymbol = page.header.activeCurrency.symbol %}
+    {% endif %}
+
+    {% set unit = currencySymbol %}
+{% endif %}
```

Comment:
I've integrated your suggested change for now, but: Your version breaks backward compatibility, since if someone sets `currencySymbol`in order to specify a unit, his setting will be not be used.

I suggest:
```
{% if unit is not defined %}
    {% set unit = page.header.activeCurrency.symbol %}
    {% if currencySymbol is defined %}
        {% set unit = currencySymbol %}
    {% endif %}
{% endif %}

{# @deprecated tag:v6.5.0 - `currencySymbol` will be removed use `unit` instead #}
{% if currencySymbol is not defined %}
    {% set currencySymbol = page.header.activeCurrency.symbol %}
{% endif %}
```
Where the `{% if currencySymbol is defined %}` block is removed when the deprecated names are removed.

Reply:
Good catch. I guess the perfect solution with that the `filter-range.plugin.js` will work as expected would be:
```
{% if unit is not defined %}
    {% set unit = page.header.activeCurrency.symbol %}
{% endif %}

{# @deprecated tag:v6.5.0 - `currencySymbol` will be removed use `unit` instead #}
{% if currencySymbol is defined %}
    {% set unit = currencySymbol %}
{% endif %}
```

Reply:
Agreed, your version is clearer than my proposal.